### PR TITLE
docs/getting-started: Add general intro to Ignition

### PR DESCRIPTION
**Pull request by joshix**
Clean up the markdown source acc to STYLE.md rules.

Supersedes #187.

### CODE REVIEW

File: doc/getting-started.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
-# Getting Started with Ignition #
+# Getting Started with Ignition
 
-Ignition uses a JSON configuration file to represent the set of changes to be made. The format of this config is detailed [in the spec][config spec]. One of the most important parts of this config is the version number. This **must** match the version number accepted by Ignition. If the config version isn't accepted by Ignition, Ignition will fail to run and prevent the machine from booting. This can be seen by inspecting the console output of the failed instance. For more information, check out the [troubleshooting section](#troubleshooting).
+*Ignition* is a low-level system configuration utility. The Ignition executable is part of the temporary initial root filesystem, the *initramfs*. When Ignition runs, it finds configuration data in a named location for a given environment, such as a file or URL, and applies it to the machine before `switch_root` is called to mount the machine's root filesystem.
```
Comment (crawford) [inline doc/getting-started.md]: "to mount the machine's root filesystem." should read "to pivot to the machine's root filesystem." The original purpose of the initramfs was to mount the root filesystem, so it should be mounted by the time the pivot occurs.

**Comment by crawford** [comment]
Small nit. Can you also rebase on top of master?

**Comment by joshix** [comment]
closed in favor of #198 because my git foo failed

**Pull request closed without merging**